### PR TITLE
use normal tags for installations

### PR DESCRIPTION
**Pull request by jamesgorrie**
## Who was this for?
Consistinites

## What is it doing for them?
Making sure we use the big tags for "Part of"

screen shot 2018-05-14 at 17 57 59

### CODE REVIEW

File: common/views/components/Templates/BasicPage/InstallationPage.js [hunk 1]
```diff
@@ -15,28 +15,23 @@ type Props = {|
...
-  const tags = [{
-    text: 'Installations'
-  }];
 
   return (
     <BasicPage
       Background={WobblyBackground()}
+      TagBar={
+        <div
+          style={{ height: '48px' }}
```
Comment (davidpmccormick) [inline common/views/components/Templates/BasicPage/InstallationPage.js]: will this be a problem if the tags break onto two lines? Maybe `min-height`?

Comment (jamesgorrie) [reply common/views/components/Templates/BasicPage/InstallationPage.js]: Done

Comment (davidpmccormick) [reply common/views/components/Templates/BasicPage/InstallationPage.js]: What's your rule of thumb for when to use inline styles?

1. when it's something that's coming from a CMS
2. when it's totally custom to the thing being styled

?

Comment (jamesgorrie) [reply common/views/components/Templates/BasicPage/InstallationPage.js]: > when it's totally custom to the thing being styled

When we use it nowhere else, and can't see the use of it anywhere else.
A bad example was when I did `align-items` - the class made more sense.

**Pull request merged**